process_vanguard_bibs.py

```diff
@@ -97,12 +98,54 @@ def move_939_fatadb(record):
         record.remove_field(old_fld)
     for fld in record.get_fields("939"):
         fld.tag = "969"
+        record.remove_field(fld)
         record.add_ordered_field(fld)
 
 def do_SILSLA_15_bib(record, dbcode):
     delete_various_9xx(record)
     copy_001(record, dbcode)
     move_9xx(record)
+
+#SILSLA-16
+
+def delete_035_subfield(record):
+    """Delete 035 $9 where appropriate"""
+    for fld in record.get_fields("035"):
+        if fld["9"] != None and fld["9"] == 'ExL' and fld["a"] != None:
```

Comment:
For comparisons to `None`, PEP style guide (and other resources) say to use `Is` or `Is Not`, instead of `==` or `!=`.
https://legacy.python.org/dev/peps/pep-0008/#programming-recommendations for example.

Please change this here and in other places.